arango/collection.py

```diff
@@ -753,6 +756,7 @@ def find(
         skip: Optional[int] = None,
         limit: Optional[int] = None,
         allow_dirty_read: bool = False,
+        sort: Jsons = [],
```

Comment:
```suggestion
        sort: Optional[Jsons] = None,
```

This is to fix a PEP warning. `[]` is mutable when used as default argument.